### PR TITLE
fix: update node version in workflows, update errors to error

### LINKED ISSUE #320: Update nr1-network-telemetry
### Acceptance Criteria

- [x] Upgrade to node 16
- [x] If applicable resolve any app deprecations planned for march 1st. 
	- [x] "normal" type for header controls has been deprecated. Use any of "primary", "secondary", "tertiary" instead
- [x] update .nvrmc file to Node 16
- [x] Confirm if we need to set the NODE_VERSION environment variable
- [x] Ensure all documentation is updated.

### PR DESCRIPTION
**Pull request by clarkmcadoo**
Cannot find a way to test this locally? Will talk about in Parking Lot 🅿️  tomorrow.

Closes https://github.com/newrelic/nr1-catalog/issues/320

### CODE REVIEW

**Review by jpvajda — APPROVED** [review]

**Comment by jpvajda** [comment]
This was tested using an account on staging that uses the sflow event type.

Screen Shot 2022-02-25 at 12 30 01 PM

File: nr1.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "schemaType": "NERDPACK",
-  "id": "c03725a8-878f-408b-9853-6568e0ee087a",
+  "id": "b4233aec-ad78-4cd9-b18b-c481cf4cd3b5",
```
Comment (josephgregoryii) [inline nr1.json:3]: Just double checking we want a new id here

**Comment by josephgregoryii** [comment]
Looks great, just one comment about `nr1.json` and a question about if we're adding a `.nvmrc` file

**Comment by clarkmcadoo** [comment]
Screen Shot 2022-02-25 at 12 30 01 PM

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**